testing/cypress/e2e/excludedLinks/excluded-links.cy.js

```diff
@@ -0,0 +1,16 @@
+/// <reference types="Cypress" />
+
+// this test will run out of the pipeline and will be a manaul test each sprint 
+
+const english = require("../../fixtures/sample-links-page.json");
```

Comment:
currently this is just a test json file to make sure the test is running correctly.